cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -168,6 +175,18 @@ public void execute(Project project) {
             Module module = project.currentPackage().module(moduleId);
             ModuleName moduleName = module.moduleName();
 
+            if (isModuleTestExecution) {
+                if (moduleExecTests.contains("--all")) {
```

Comment:
I don't think we are supporting "--all". So, we should be able to get rid of it. WDYT @azinneera ?

Reply:
Removed `--all` functionality